tests/integration-tests/conftest.py

```diff
@@ -436,24 +461,13 @@ def _get_arn_partition(region):
         return "aws"
 
 
-def _enable_sanity_check_if_unset(cluster_config):
-    config = configparser.ConfigParser()
-    config.read(cluster_config)
-
-    if "global" not in config:
-        config.add_section("global")
-
-    if "sanity_check" not in config["global"]:
-        config["global"]["sanity_check"] = "true"
-
-    with cluster_config.open(mode="w") as f:
-        config.write(f)
-
-
 def _get_default_template_values(vpc_stack, request):
     """Build a dictionary of default values to inject in the jinja templated cluster configs."""
     default_values = get_vpc_snakecase_value(vpc_stack)
     default_values.update({dimension: request.node.funcargs.get(dimension) for dimension in DIMENSIONS_MARKER_ARGS})
+    scheduler = default_values.get("scheduler")
+    if scheduler is not None:
+        default_values["scheduler"] = scheduler.title()
```

Comment:
I would do this in the cluster config by using jinja builtin filter: `{{ scheduler | title }}`

Reply:
Why would you prefer the code change in multiple config files than this single change?